### PR TITLE
Publish OSLC PROMCODE OS files under OSLC NS

### PR DESCRIPTION
**Pull request by tomkamimura**
I have modified promcode-shapes.ttl and promcode-vocab.ttl files per the comments of Andrew and Jim.

### CODE REVIEW

File: content/ns/promcode/promcode-shapes.ttl [hunk 1]
```diff
@@ -35,7 +35,7 @@
   dcterms:license <http://www.apache.org/licenses/LICENSE-2.0> ;
   dcterms:source <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/promcode-shapes.ttl> ;
   dcterms:isPartOf <https://tools.oasis-open.org/version-control/browse/wsvn/oslc-promcode/shape/trunk/spec.html> ;
-  dcterms:hasVersion "CS02" .
+  dcterms:hasVersion "CS03" .
```
Comment (berezovskyi) [inline content/ns/promcode/promcode-shapes.ttl:38]: Why CS03?

**Comment by tomkamimura** [comment]
Andrew,

Sorry, it is CS02. I will push the changed files.

Tom

File: content/ns/promcode/promcode-shapes.ttl (hunk 1 above)
Comment (tomkamimura) [reply content/ns/promcode/promcode-shapes.ttl]: Sorry, it should be CS02. The latest version has corrected it.

**Comment by berezovskyi** [comment]
I have provided my latest comment here: https://github.com/oslc-op/website/issues/278#issuecomment-1043113242

**Comment by tomkamimura** [comment]
promcode-shapes.ttl and promcode.vocab.ttl files are updated on February 24, 2022. See the comment at https://github.com/oslc-op/website/issues/278

**Comment by tomkamimura** [comment]
Andrew,

I posted two comments into #278 at https://github.com/oslc-op/website/issues/278 <https://github.com/oslc-op/website/issues/278> . Please take a look.

Tom Kamimura

**Comment by tomkamimura** [comment]
I ran ShapeChecker again on the lastest promcode-shapes.ttl and promcode-voca.ttl today (March 2, 2022). One warning generated which says that shapes.ttl does not refernce voca.ttl. Here is the result.

Results from ShapeChecker run on Wed Mar 02 15:16:52 JST 2022

Command line arguments:
-t Warning -C -q EmbeddedWhitespace -q MissingPeriod -x http://open-services.net/ns/promcode -x
http://open-services.net/ns/promcode#.* -x http://purl.org/dc/terms/ -x http://purl.org/dc/terms/.* -v
/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl -s
../../../turtle_files/promcode-shapes.ttl

A total of 1 issue was found (0 info, 1 warnings, 0 errors)

Checked vocabulary file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Checked shapes from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/build/install/ShapeChecker/../../../turtle_files/promcode-shapes.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Warning: This vocabulary was given, but not referenced in the given shapes:
file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl

**Comment by tomkamimura** [comment]
We looked at our turtle files and our script to run ShapeCheck. There were some blank characters in places that may have caused a warning. Also, we changed parameters of the script to detect such blank characters. After fixing those blank characters, we ran the script and a warning is now gone. Here is the output of ShapeCheck. So, I think we are good now. I have modified the turtle files in the active PQ. @berezovskyi, will you please move ahead to get an agreement from oslc-ops?

Command line arguments:
-t Warning -C -q unusedVocabulary -x http://open-services.net/ns/promcode -x http://open-services.net/ns/promcode#.* -x
http://purl.org/dc/terms/ -x http://purl.org/dc/terms/.* -v
/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl -s
../../../turtle_files/promcode-shapes.ttl

A total of 0 issues were found (0 info, 0 warnings, 0 errors)

Checked vocabulary file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/turtle_files/promcode-vocab.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

Checked shapes from file:/Users/tomkamimura/PROMCODE/oslc-promcode/PROMCODE/build/install/ShapeChecker/../../../turtle_files/promcode-shapes.ttl, with 0 issues (0 info, 0 warnings, 0 errors)

**Comment by tomkamimura** [comment]
One more comment. We double checked our turtle files and confirmed the all the terms defined in vocab.ttl are referenced in shapes.ttl. You might think that a warning is gone because we added the parameter -q unusedVocabulary. We tested turtle files of QM that does not produce any warning. The script of Qm uses this parameter also. But, when we removed this parameter in the script of QM, it produced the same warning as the one we used to get. So, I don't know what causes a warning when the parameter unusedVocabulary is not used. At least, we can say that if there is a problem, QM probably has the same problem. In any case, the warning message produced by ShapeCheck does not seem accurate.

**Comment by berezovskyi** [comment]
@tomkamimura getting ready to merge and publish. Could you please check the files again, please? I saw on the lists that PROMCODE hit the OASIS Standard stage, but the files still say CS03. Should we change those to OS before publication?

**Comment by tomkamimura** [comment]
Andrew,

Thank you for your help. Yes, PROMCODE is now an OASIS standard, and that means that editorial change will be needed. I will check the files and will make necessary change before they are merged.

Tom Kamimura

**Comment by tomkamimura** [comment]
Andrew, I have updated shapes.ttl and vocab.ttl files that rerflect the OASIS Standard version.

**Comment by tomkamimura** [comment]
@berezovskyi , the correct date is March 11, 2022. I changed the wrong date in some files to 2022-03-11.

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**